evaluate/task/task-write-test.go

```diff
@@ -52,11 +52,6 @@ func (t *TaskWriteTests) Run(repository evaltask.Repository) (repositoryAssessme
 	}
 	defer logClose()
 
-	log.Printf("Evaluating model %q on task %q using language %q and repository %q", t.Model.ID(), t.Identifier(), t.Language.ID(), repository.Name())
-	defer func() {
-		log.Printf("Evaluated model %q on task %q using language %q and repository %q: encountered %d problems: %+v", t.Model.ID(), t.Identifier(), t.Language.ID(), repository.Name(), len(problems), problems)
-	}()
-
```

Comment:
Would be cool if we could also include the log file creation, and really move it into a helper cause the evaluation is already soooo cluttered...

Reply:
Follow-up